django_tasks/backends/database/models.py

```diff
@@ -170,6 +170,13 @@ def task_result(self) -> "TaskResult[T]":
 
         return task_result
 
+    @property
+    def task_name(self) -> str:
+        try:
+            return self.task_path.rsplit(".", 1)[1]
+        except IndexError:
```

Comment:
Shouldn't ever happen (a function must be in a module, right?), but adding it for safety.

Reply:
Question: Is there a reason to move this onto the model vs keeping it in the admin?

Reply:
A combination of it feeling like a fairly simple model/object property, and it being slightly easier to test methods on models.

Happy to switch back if you'd prefer.

Reply:
Keeping logic out the admin interface probably makes sense, so leaving it here is probably fine. `DBTaskResult` isn't a public API, so it's not the end of the world if we need to move methods around.